hack/ukify/main.go

```diff
@@ -165,6 +165,8 @@ func Measure(tempDir, kernel, signingKey string, sections []section) ([]section,
 func run() error {
 	defaultCmdline := procfs.NewCmdline("")
 	defaultCmdline.Append(talosconstants.KernelParamPlatform, "metal")
+	defaultCmdline.Append("console", "ttyS0)")
```

Comment:
```suggestion
	defaultCmdline.Append("console", "ttyS0")
```

Reply:
and it also makes sense to append first these: https://github.com/siderolabs/talos/blob/f0fadcc9cd45bb077e73d905a74023311c500732/pkg/machinery/kernel/kernel.go#L18-L33